tfhe/docs/fundamentals/compress.md

```diff
@@ -41,6 +41,84 @@ fn main() {
 }
 ```
 
+## Ciphertexts compressed while encrypted
```

Comment:
Ciphertext == encrypted, so rather "Ciphertexts compressed after computations" for example

Reply:
Done